@stellar/design-system/src/components/Typography/styles.scss

```diff
@@ -0,0 +1,145 @@
+@use "../../utils.scss" as *;
+
+.Heading {
+  font-family: var(--font-family-base);
+  font-weight: var(--font-weight-regular);
+  letter-spacing: 0;
+
+  &--xxl {
+    font-size: pxToRem(40px);
```

Comment:
`pxToRem` method in action. It _should_ work everywhere, but we'll make sure once we use it in more places.